bambi/backend/pymc.py

```diff
@@ -424,21 +432,5 @@ def _get_bayeux_methods():
 
```

Comment:
Probably want to return `{"mcmc": []}` on line 431 to avoid `KeyError` elsewhere.